src/client/components/pages/entities/cb-review.js

```diff
@@ -0,0 +1,70 @@
+import * as bootstrap from 'react-bootstrap';
+const { Col, Row } = bootstrap;
+import React from 'react';
+import { Rating } from "react-simple-star-rating";
+
+
+function ReviewCard({ reviewData }) {
+    const publishedDate = new Date(reviewData.published_on).toDateString();
+    let reviewText = reviewData.text;
+    if (reviewText.length > 75) {
+        reviewText = reviewText.substring(0, 75) + '...';
+    }
+    const reviewLink = "https://critiquebrainz.org/review/" + reviewData.id;
+    return (
+        <div className='border bg-light p-2 mb-2' >
+            <div>
+                <Rating
+                    className="rating-stars"
+                    ratingValue={0}
+                    initialValue={reviewData.rating}
+                    size={15}
+                    stars={5}
+                    readonly
+                    allowHalfIcon
+                    fillColor="#46433A"
+                    allowHover={false}
+                />
+                <small className='float-right'>
+                    Review by: <b>{reviewData.user.display_name}</b> {publishedDate}
+                </small>
+            </div>
+            <div>
+                {reviewText}
+                <a className='float-right' href={reviewLink}>View &gt;</a>
+            </div>
+        </div>
+    )
+}
+
+function EntityReviews({ entity }) {
+    const reviews = entity.reviews.reviews;
+    let reviewContent;
+    const mapEntityType = {
+		EditionGroup: "bb_edition_group",
+	};
+    let entityType = mapEntityType[entity.type];
```

Comment:
~~This code is duplicated; it would be good to make it a helper (we won't win a prize for the name, but what about `getReviewEntityNameFromType` or `getCritiqueBrainzEntityType`?) in `scr/common/…` that can be used both by the server and client side.~~

Edit: With my comment below about kebabCase I don't think this will be necessary

Reply:
We would have to keep this, because the links in CB are not always the kebabCase links returned by the defined function in utils. Like for the entity `Work` , we don't have the link critiquebrainz.org/work/\<entity-id> for it.